components/datepicker/DatePickerInputRange.js

```diff
@@ -232,11 +231,7 @@ class DatePickerInputRange extends PureComponent {
           {this.renderIcon()}
           {this.renderDayPickerInput()}
         </div>
-        {error ? (
-          <ErrorText inverse={inverse}>{error}</ErrorText>
-        ) : (
-          helpText && <HelpText inverse={inverse}>{helpText}</HelpText>
-        )}
+        <ValidationText error={error} help={helpText} inverse={inverse} />
```

Comment:
In all inputs there is somewhat a mismatch in naming for the `error` and `helpText` props, which are very similar, making their naming consistent didn't feel part of this PR for me (it would also introduce a breaking change.)